lib/Charts/LineChart.js

```diff
@@ -287,16 +296,18 @@ function render(element, state) {
     if (state.mini) {
```

Comment:
The reason I kept it is that I saw in this PR this line has been removed: 💃 https://github.com/TerriaJS/terriajs/pull/1874/files